candi/bashible/common-steps/cluster-bootstrap/070_install_control_plane.sh.tpl

```diff
@@ -12,10 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{{- $experimentalOption := "" -}}
-{{- if semverCompare "<1.22" .kubernetesVersion -}}
-  {{- $experimentalOption = "--experimental-patches /var/lib/bashible/kubeadm/patches" -}}
-{{- end }}
+{{- $experimentalOption := "" }}
+
```

Comment:
experimentalOption is not needed anymore because it is always empty

Reply:
yup. But it appends to the next commands. It would be just an empty string and we don't have to clean up all these commands. Also it's a base for some options in the future if we need them

Reply:
okayyyy